docs/README_EN.md

```diff
@@ -10,7 +10,7 @@
   <img src="https://img.shields.io/badge/first_contributions-%F0%9F%94%B0-white?style=flat" alt="Static Badge">
 </p>
 
-[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twttr-share]
+![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)
```

Comment:
This code, in its previous state, is fine.

As it is now, the image is displayed and we need to treat it as a Link.
[ref](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links)